lib/tools/apk-utils.js

```diff
@@ -1,4 +1,4 @@
-import { buildStartCmd, getPossibleActivityNames } from '../helpers.js';
```

Comment:
Since `getPossibleActivityNames` is no longer used here, it needs to be pulled from the codebase altogether. This includes in `lib/helpers.js` and `test/unit/helper-specs.js`.